vue-components/src/components/OptionsMenu.vue

```diff
@@ -18,6 +20,9 @@
 			@mousedown.prevent="activeItemIndex = index"
 			@mouseup="activeItemIndex = -1"
 			ref="menu-items"
+			role="option"
+			:aria-label="`${menuItem.label} - ${menuItem.description}`"
```

Comment:
Have you tried putting the description into [`aria-description`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-description)?

Reply:
much better. thanks for the suggestion.

Reply:
this works when testing orca locally but later i discovered it creates the a11y checks to fail :( `role=option` is not compatible with `aria-description`. you can see the error here: https://github.com/wmde/wikit/runs/6597303500?check_suite_focus=true#step:5:193

Reply:
Ok, this is strange. It feels almost like an oversight, given that `aria-describedby` _is_ allowed in https://www.w3.org/TR/wai-aria-1.1/#option.

But we can work around this. Can you give an idea to the field with the description for each option and reference that in `aria-describedby`?

Reply:
That's a very good workaround! Thanks